docs/reference/changelog.rst

```diff
@@ -22,6 +24,34 @@ Documentation:
 
 - Correct the Maven plugin docstring to refer to Maven from Go.
 
+2.3.0 (unreleased)
```

Comment:
I believe this changelog is wrong now with regards to the *other* items because we had a recent issue/mistake with it. Can you compare this with `main` and update please?